nbextensions/appCell2/widgets/appCellWidget-fsm.js

```diff
@@ -637,7 +637,7 @@ define([], () => {
                     ],
                 },
             },
-            next: [STATE.TERMINATED, STATE.EDITING_COMPLETE],
+            next: [STATE.TERMINATED, STATE.EDITING_COMPLETE, STATE.INTERNAL_ERROR],
```

Comment:
spotted a couple of missing states during my app cell testing

Reply:
😖